### ISSUE TITLE
Cannot load previous model in RC2

### ISSUE THREAD

**Issue by xiangyang3131777** [Microsoft/CNTK#1788]
I cannot use “restore_from_checkpoint” to load the model in RC2, Which I utilized "save_checkpoint"  to saved in rc1...........................................

**Comment by cha-zhang** [comment]
Could you let us know what kind of model do you have?

**Comment by xiangyang3131777** [comment]
My model was defined as follow:

############################################################
hidden_layers_dim = 2048
input_dim=129*11
output_dim=129
dnn_input=C.input(input_dim)
dnn_output=C.input(output_dim)

def create_DNN_model(features):
    h=features
    h=Dense(hidden_layers_dim,activation=C.ops.sigmoid)(h)
    h = C.layers.Dropout(0.2)(h)
    h=Dense(hidden_layers_dim,activation=C.ops.sigmoid)(h)
    h = C.layers.Dropout(0.2)(h)
    h=Dense(hidden_layers_dim,activation=C.ops.sigmoid)(h)
    h = C.layers.Dropout(0.2)(h)
    prediction=Dense(output_dim,activation=C.ops.leaky_relu)(h)
    return prediction

dnn_z=create_DNN_model(dnn_input)
dnn_loss = C.squared_error(dnn_z,dnn_output)
dnn_error=C.squared_error(dnn_z,dnn_output)
learning_rate=0.045
minibatch_size = 128

lr_schedule = learning_rate_schedule(learning_rate, UnitType.minibatch) 
dnn_learner=sgd(dnn_z.parameters,lr_schedule)
dnn_trainer=Trainer(dnn_z,(dnn_loss,dnn_error),[dnn_learner])
.......
.......
........
dnn_trainer.save_checkpoint("lg_t.dnn")  
############################################################

**Comment by eldakms** [comment]
Closing due to inactivity. Feel free to reopen if the problem is reproducible on the latest binaries. Thanks!
